### DISCUSSION TITLE
Portfólio - Uma análise de Desenvolvedores

### DISCUSSION THREAD

**Discussion by solilorys** [frontendbr/forum#2338] [Dicas]
Olá pessoal!

Sou iniciante como desenvolvedora e queria algumas dicas e conselhos de vocês.
Fiz recentemente meu portfólio apenas com HTML, CSS e JS, mas por eu ser iniciante sei que deve ter muita coisa desorganizada, falta de boas práticas, entre outras coisas. Queria que vocês, se pudessem, me ajudassem a encontrar e entender esses erros para eu melhorar meu portfólio e meus conhecimentos, pois eu não vejo nada demais, mas desenvolvedores experientes sim! Então me da uma força e ajuda 

Meu projeto ta no meu repositório "portfolio"

Obrigada C:

**Comment by felquis** [comment]
Link https://solilorys.github.io/portfolio/

**Comment by felquis** [comment]
Laryssa, seu site está lindo, parabéns!

**Falando de HTML**, talvez vc queira experimentar uma extensão chamada "HTML5 Outliner", no seu site mostra que as sessões estão usando títulos corretamente, com algumas pequenas excessões fáceis de corrigir. Essa extensão me ajuda muito a muito tempo. a `<nav>` a nav ou body é aceitável ficar sem title na minha experiência, mas outras ocasiões geralmente há alternativas melhores.

![Captura de Tela 2022-08-23 às 10 09 01](https://user-images.githubusercontent.com/736728/186166139-3301839a-16f6-463c-ab0e-ce2602edf4b7.png)

-----

Também, nessa sessão não entendi que tratava-se de tabs, talvez queira melhorar a UI/UX para se parecerem mais com tabs?
Mas essa observação é sobre os elementos usados, pois apesar de ser algo feito para ser clicável, vc não usou um `<a>` ou `<button>` e sim uma `<div>` o que não é recomendado num geral, se algo da pra clicar e afeta a navegação geralmente deve ser usado um `a` ou `button`.

![Captura de Tela 2022-08-23 às 10 12 21](https://user-images.githubusercontent.com/736728/186166999-d26f5097-b518-4e70-9757-8ce67ce40cac.png)


----

De forma geral, o "DevTools Lighthouse" pode te dar dicas que passam despercebidos por nós, não tem como dar dica de tudo, e não da para corrigir tudo que ele sugere, mas pode ser uma fonte de descobertas interessantes que podemos fazer.
![Captura de Tela 2022-08-23 às 10 22 12](https://user-images.githubusercontent.com/736728/186169149-3f2c2479-27ad-4eac-93ae-91109aed1a1e.png)

-----

Como você está usando unidades rem, que particularmente também uso extensivamente, é interessante vc testar seu site em tamanhos de fonte diferentes pois alguns elementos podem quebrar, trazendo este exemplo da sessão de habilidades. No design responsivo, raramente usamos height definido. Geralmente o conteúdo é que dita o tamanho do height. E este é um bom exemplo para a vida.
![Captura de Tela 2022-08-23 às 10 27 29](https://user-images.githubusercontent.com/736728/186170260-9de2c1a9-57c6-4d1e-bfea-2f08e4ddedf2.png)


-----

Senti falta do menu do header no celular, mas a dica acima também serve para vc poder verificar se o menu do header também fica bem com fontes maiores. O que acontece, é que em celulares android e ios, tem uma opção para aumentar o tamanho da fonte do sistema. Então é interessante você testar com algumas das opções de fontes que estes sistemas dão aos usuários, geralmente, as pessoas maiores de 50 anos que conheço todas usam uma fonte maior. Então você não precisa testar na fonte maior de todas, mas em algo razoavelmente maior. No iOS este recurso tem um limite bem claro, já no Android não me lembro agora. Enfim, divirta-se.

----

Outra dica legal, é revisar as meta tags de compartilhamento, pois a experiência de compartilhar no Instagram/Facebook/Whatsapp usam Open Graph metatags, o facebook da uma ferramenta para testar quais dados ele usará para exibir seu site https://developers.facebook.com/tools/debug/ e no whatsapp é um pouco diferente, talvez seja algo que você queira melhorar em praticamente qualquer site que você faça. O twitter tem o Twitter Cards https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup e enfim, é uma dica legal.

![Captura de Tela 2022-08-23 às 10 34 24](https://user-images.githubusercontent.com/736728/186171858-bb3ade58-5f21-4731-8923-fb0dc477cce0.png)


-----

Agora uma outra dica, é vi que você deixou seu número de celular, e email. O que está tudo bem. Porém, se você tem conta em bancos que usam este celular e/ou e-mail, é interessante você não colocar em seu site em minha opinião, e nem em outros sites. E claro, seu CPF jamais né, mas talvez nos dias de hj, talvez seja interessante não deixar o número/email aberto. É algo que venho reavaliando para mim, não tenho certeza da efetividade, mas deixo aqui a sugestão para você também.

----

É isso que pensei em dizer, espero que algo possa ser útil. Me segue no Twitter/Github, vlw.

**Reply by solilorys** [reply]
Ajudou demais!! Vou melhorar tudo que você disse e um pocuos mais!

Muito obrigada mesmo, me ajudou pra caramba.
